larq_compute_engine/mlir/python/converter.py

```diff
@@ -121,11 +200,30 @@ def convert_keras_model(
             "This should only be used for latency tests."
         )
     if hasattr(model, "dtype_policy") and model.dtype_policy.name != "float32":
-        raise RuntimeError(
+        raise ValueError(
             "Mixed precision float16 models are not supported by the TFLite converter, "
             "please convert them to float32 first. See also: "
             "https://github.com/tensorflow/tensorflow/issues/46380"
         )
+
+    # First attempt conversion as saved model
+    try:
+        with tempfile.TemporaryDirectory() as saved_model_dir:
+            model.save(saved_model_dir, save_format="tf")
+
+            return convert_saved_model(
+                saved_model_dir,
+                inference_input_type=inference_input_type,
+                inference_output_type=inference_output_type,
+                experimental_default_int8_range=experimental_default_int8_range,
+                experimental_enable_bitpacked_activations=experimental_enable_bitpacked_activations,
+                target=target,
+            )
+    except Exception:
+        warnings.warn(
+            "Saved-model conversion failed, falling back to graphdef-based conversion."
+        )
```

Comment:
This is now enabled by default. Let me know if you prefer to keep this behind a flag instead.